2025-05-19 18:24:12,695 WARNING: PyNaCl is not installed, voice will NOT be supported
2025-05-19 18:24:12,749 INFO: logging in using static token
2025-05-19 18:24:13,369 INFO: Shard ID None has sent the IDENTIFY payload.
2025-05-19 18:24:13,584 INFO: Shard ID None has connected to Gateway: ["gateway-prd-us-east1-b-p82x",{"micros":72160,"calls":["id_created",{"micros":409,"calls":[]},"session_lookup_time",{"micros":337,"calls":[]},"session_lookup_finished",{"micros":15,"calls":[]},"discord-sessions-prd-2-10",{"micros":70917,"calls":["start_session",{"micros":65498,"calls":["discord-api-rpc-6c744fbcb7-886nm",{"micros":57050,"calls":["get_user",{"micros":8221},"get_guilds",{"micros":16235},"send_scheduled_deletion_message",{"micros":10},"guild_join_requests",{"micros":1},"authorized_ip_coro",{"micros":9},"pending_payments",{"micros":926},"apex_user_experiments",{"micros":2},"user_activities",{"micros":1}]}]},"starting_guild_connect",{"micros":62,"calls":[]},"presence_started",{"micros":336,"calls":[]},"guilds_started",{"micros":65,"calls":[]},"lobbies_started",{"micros":1,"calls":[]},"guilds_connect",{"micros":1,"calls":[]},"presence_connect",{"micros":4939,"calls":[]},"connect_finished",{"micros":4943,"calls":[]},"build_ready",{"micros":9,"calls":[]},"clean_ready",{"micros":1,"calls":[]},"optimize_ready",{"micros":0,"calls":[]},"split_ready",{"micros":1,"calls":[]}]}]}] (Session ID: 88201b847cd2263c2a489219f22acbf7).
2025-05-19 18:24:24,089 INFO: Received signal to terminate bot and event loop.
2025-05-19 18:24:24,089 INFO: Cleaning up tasks.
2025-05-19 18:24:24,089 INFO: Cleaning up after 3 tasks.
2025-05-19 18:24:24,090 INFO: All tasks finished cancelling.
2025-05-19 18:24:24,090 INFO: Closing the event loop.
2025-05-19 18:25:26,231 WARNING: PyNaCl is not installed, voice will NOT be supported
2025-05-19 18:25:26,234 INFO: logging in using static token
2025-05-19 18:25:26,722 INFO: Shard ID None has sent the IDENTIFY payload.
2025-05-19 18:25:27,153 INFO: Shard ID None has connected to Gateway: ["gateway-prd-us-east1-d-20n4",{"micros":299957,"calls":["id_created",{"micros":609,"calls":[]},"session_lookup_time",{"micros":4548,"calls":[]},"session_lookup_finished",{"micros":19,"calls":[]},"discord-sessions-prd-2-48",{"micros":294504,"calls":["start_session",{"micros":276578,"calls":["discord-api-rpc-6c744fbcb7-2sb7p",{"micros":263429,"calls":["get_user",{"micros":76530},"get_guilds",{"micros":47402},"send_scheduled_deletion_message",{"micros":12},"guild_join_requests",{"micros":1},"authorized_ip_coro",{"micros":10},"pending_payments",{"micros":2202},"apex_user_experiments",{"micros":5},"user_activities",{"micros":2}]}]},"starting_guild_connect",{"micros":57,"calls":[]},"presence_started",{"micros":3815,"calls":[]},"guilds_started",{"micros":61,"calls":[]},"lobbies_started",{"micros":1,"calls":[]},"guilds_connect",{"micros":1,"calls":[]},"presence_connect",{"micros":13972,"calls":[]},"connect_finished",{"micros":13976,"calls":[]},"build_ready",{"micros":13,"calls":[]},"clean_ready",{"micros":1,"calls":[]},"optimize_ready",{"micros":0,"calls":[]},"split_ready",{"micros":1,"calls":[]}]}]}] (Session ID: e92798b29be536e153314c157eec1ae2).
2025-05-19 18:25:38,595 INFO: HTTP Request: POST https://openrouter.ai/api/v1/chat/completions "HTTP/1.1 200 OK"
2025-05-19 18:26:04,950 INFO: HTTP Request: POST https://openrouter.ai/api/v1/chat/completions "HTTP/1.1 200 OK"
2025-05-19 18:26:18,004 WARNING: Shard ID None heartbeat blocked for more than 10 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 313, in extract_scheduled_commands
    admin_result = await self.chat_with_deepseek(admin_prompt, 'admin')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:26:28,019 WARNING: Shard ID None heartbeat blocked for more than 20 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 313, in extract_scheduled_commands
    admin_result = await self.chat_with_deepseek(admin_prompt, 'admin')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:26:38,020 WARNING: Shard ID None heartbeat blocked for more than 30 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 313, in extract_scheduled_commands
    admin_result = await self.chat_with_deepseek(admin_prompt, 'admin')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:26:45,764 INFO: HTTP Request: POST https://openrouter.ai/api/v1/chat/completions "HTTP/1.1 200 OK"
2025-05-19 18:26:48,026 WARNING: Shard ID None heartbeat blocked for more than 40 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 325, in extract_scheduled_commands
    eventer_result = await self.chat_with_deepseek(eventer_prompt, 'eventer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:26:58,030 WARNING: Shard ID None heartbeat blocked for more than 50 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 325, in extract_scheduled_commands
    eventer_result = await self.chat_with_deepseek(eventer_prompt, 'eventer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:27:08,042 WARNING: Shard ID None heartbeat blocked for more than 60 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 325, in extract_scheduled_commands
    eventer_result = await self.chat_with_deepseek(eventer_prompt, 'eventer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:27:18,044 WARNING: Shard ID None heartbeat blocked for more than 70 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 325, in extract_scheduled_commands
    eventer_result = await self.chat_with_deepseek(eventer_prompt, 'eventer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:27:28,054 WARNING: Shard ID None heartbeat blocked for more than 80 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 325, in extract_scheduled_commands
    eventer_result = await self.chat_with_deepseek(eventer_prompt, 'eventer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:27:38,060 WARNING: Shard ID None heartbeat blocked for more than 90 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 325, in extract_scheduled_commands
    eventer_result = await self.chat_with_deepseek(eventer_prompt, 'eventer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:27:43,423 INFO: HTTP Request: POST https://openrouter.ai/api/v1/chat/completions "HTTP/1.1 200 OK"
2025-05-19 18:27:48,069 WARNING: Shard ID None heartbeat blocked for more than 100 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 337, in extract_scheduled_commands
    designer_result = await self.chat_with_deepseek(designer_prompt, 'designer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:27:58,084 WARNING: Shard ID None heartbeat blocked for more than 110 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 337, in extract_scheduled_commands
    designer_result = await self.chat_with_deepseek(designer_prompt, 'designer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:28:08,094 WARNING: Shard ID None heartbeat blocked for more than 120 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 337, in extract_scheduled_commands
    designer_result = await self.chat_with_deepseek(designer_prompt, 'designer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:28:18,105 WARNING: Shard ID None heartbeat blocked for more than 130 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 337, in extract_scheduled_commands
    designer_result = await self.chat_with_deepseek(designer_prompt, 'designer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:28:28,118 WARNING: Shard ID None heartbeat blocked for more than 140 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 337, in extract_scheduled_commands
    designer_result = await self.chat_with_deepseek(designer_prompt, 'designer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:28:38,129 WARNING: Shard ID None heartbeat blocked for more than 150 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1217, in on_interaction
    await self.process_application_commands(interaction)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 871, in process_application_commands
    await self.invoke_application_command(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\bot.py", line 1137, in invoke_application_command
    await ctx.command.invoke(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 435, in invoke
    await injected(ctx)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 138, in wrapped
    ret = await coro(arg)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\commands\core.py", line 1078, in _invoke
    await self.callback(self.cog, ctx, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 206, in ai_iter
    await self.append_scheduled_commands()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 303, in __call__
    return await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 281, in append_scheduled_commands
    await self.extract_scheduled_commands(result['json_data'])
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 337, in extract_scheduled_commands
    designer_result = await self.chat_with_deepseek(designer_prompt, 'designer')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:28:38,342 INFO: Timed out receiving packet. Attempting a reconnect.
2025-05-19 18:28:38,342 INFO: Got a request to RESUME the websocket.
2025-05-19 18:28:38,670 INFO: 📁 Категорію `administrative` створено.
2025-05-19 18:28:38,677 INFO: Shard ID None has sent the RESUME payload.
2025-05-19 18:28:38,814 INFO: Shard ID None session has been invalidated.
2025-05-19 18:28:38,935 INFO: Канал `rules` створено.
2025-05-19 18:28:38,953 INFO: Got a request to IDENTIFY the websocket.
2025-05-19 18:28:39,221 INFO: ✅ Повідомлення надіслано до каналу `rules`.
2025-05-19 18:28:39,441 INFO: 📂 Канал `rules` переміщено до категорії `administrative`.
2025-05-19 18:28:40,652 INFO: Канал `welcome` створено.
2025-05-19 18:28:41,071 INFO: ✅ Повідомлення надіслано до каналу `welcome`.
2025-05-19 18:28:41,393 INFO: Канал `suggestions` створено.
2025-05-19 18:28:41,620 INFO: ✅ Повідомлення надіслано до каналу `suggestions`.
2025-05-19 18:28:41,982 INFO: Канал `ask_anything` створено.
2025-05-19 18:28:42,296 INFO: ℹ️ Інформаційний канал `introduce_yourself` створено.
2025-05-19 18:28:42,302 INFO: ❌ Канал `ask_anything_channel_id` не знайдено.
2025-05-19 18:28:42,308 INFO: ❌ Канал `introduce_yourself_channel_id` не знайдено.
2025-05-19 18:28:42,312 INFO: Канал `1374033689477910582` не знайдено або він не текстовий.
2025-05-19 18:28:42,580 INFO: Канал `загальний` видалено.
2025-05-19 18:28:42,841 INFO: Канал `rules` створено.
2025-05-19 18:28:42,846 INFO: ❌ Канал або категорію не знайдено.
2025-05-19 18:28:43,095 INFO: ℹ️ Інформаційний канал `rules_2` створено.
2025-05-19 18:28:43,099 INFO: ❌ Канал або категорію не знайдено.
2025-05-19 18:28:44,205 INFO: Shard ID None has sent the IDENTIFY payload.
2025-05-19 18:28:44,422 INFO: Shard ID None has connected to Gateway: ["gateway-prd-us-east1-d-rq5f",{"micros":75809,"calls":["id_created",{"micros":378,"calls":[]},"session_lookup_time",{"micros":3050,"calls":[]},"session_lookup_finished",{"micros":17,"calls":[]},"discord-sessions-prd-2-3",{"micros":71765,"calls":["start_session",{"micros":44622,"calls":["discord-api-rpc-6c744fbcb7-glwkf",{"micros":37042,"calls":["get_user",{"micros":6457},"get_guilds",{"micros":2836},"send_scheduled_deletion_message",{"micros":9},"guild_join_requests",{"micros":652},"authorized_ip_coro",{"micros":8},"pending_payments",{"micros":913},"apex_user_experiments",{"micros":2},"user_activities",{"micros":1}]}]},"starting_guild_connect",{"micros":52,"calls":[]},"presence_started",{"micros":11033,"calls":[]},"guilds_started",{"micros":59,"calls":[]},"lobbies_started",{"micros":1,"calls":[]},"guilds_connect",{"micros":1,"calls":[]},"presence_connect",{"micros":15978,"calls":[]},"connect_finished",{"micros":15982,"calls":[]},"build_ready",{"micros":13,"calls":[]},"clean_ready",{"micros":0,"calls":[]},"optimize_ready",{"micros":1,"calls":[]},"split_ready",{"micros":0,"calls":[]}]}]}] (Session ID: d0b75191606861d699ab04d075b6bc47).
2025-05-19 18:29:22,391 INFO: [view_scheduled] 🔍 Немає запланованих завдань для гільдії 1374033689477910578
2025-05-19 18:31:54,443 INFO: HTTP Request: POST https://openrouter.ai/api/v1/chat/completions "HTTP/1.1 200 OK"
2025-05-19 18:32:15,502 WARNING: Shard ID None heartbeat blocked for more than 10 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 182, in on_message
    await self.send_message_to_moderator(msg.content, msg.author.global_name, msg.author.id)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 189, in send_message_to_moderator
    moderator_result = await self.chat_with_deepseek(
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:32:25,511 WARNING: Shard ID None heartbeat blocked for more than 20 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 412, in _run_event
    await coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 182, in on_message
    await self.send_message_to_moderator(msg.content, msg.author.global_name, msg.author.id)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 189, in send_message_to_moderator
    moderator_result = await self.chat_with_deepseek(
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:32:30,423 INFO: Websocket closed with 1000, cannot reconnect.
2025-05-19 18:32:30,423 ERROR: Attempting a reconnect in 1.45s
Traceback (most recent call last):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 617, in connect
    await self.ws.poll_event()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\gateway.py", line 630, in poll_event
    raise ConnectionClosed(
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
2025-05-19 18:32:31,797 INFO: 📨 Повідомлення надіслано користувачу `658217734814957578`.
2025-05-19 18:32:32,129 INFO: Shard ID None has sent the RESUME payload.
2025-05-19 18:32:32,306 INFO: Shard ID None has successfully RESUMED session d0b75191606861d699ab04d075b6bc47 under trace ["gateway-prd-us-east1-b-tf27",{"micros":41144,"calls":["id_created",{"micros":0,"calls":[]},"session_lookup_time",{"micros":356,"calls":[]},"session_lookup_finished",{"micros":12,"calls":[]},"discord-sessions-prd-2-3",{"micros":27}]}].
2025-05-19 18:33:02,410 INFO: HTTP Request: POST https://openrouter.ai/api/v1/chat/completions "HTTP/1.1 200 OK"
2025-05-19 18:33:13,485 INFO: HTTP Request: POST https://openrouter.ai/api/v1/chat/completions "HTTP/1.1 200 OK"
2025-05-19 18:33:23,394 WARNING: Shard ID None heartbeat blocked for more than 10 seconds.
Loop thread traceback (most recent call last):
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\main.py", line 20, in <module>
    bot.run(Token)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 772, in run
    loop.run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\windows_events.py", line 321, in run_forever
    super().run_forever()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 607, in run_forever
    self._run_once()
  File "C:\Program Files\Python311\Lib\asyncio\base_events.py", line 1922, in _run_once
    handle._run()
  File "C:\Program Files\Python311\Lib\asyncio\events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\ext\tasks\__init__.py", line 169, in _loop
    await self.coro(*args, **kwargs)
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 352, in check_scheduled_commands
    await execute_command(self, guild, command)  # Виклик з окремого файлу
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\execute_command.py", line 247, in execute_command
    hr_result = await self.chat_with_deepseek(hr_prompt, 'hr')
  File "C:\Users\minesko\Documents\AI_Owner_repo\Dev_is_Art_bot\cogs\ai_control.py", line 412, in chat_with_deepseek
    completion = client.chat.completions.create(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_utils\_utils.py", line 287, in wrapper
    return func(*args, **kwargs)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\resources\chat\completions\completions.py", line 925, in create
    return self._post(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 1239, in post
    return cast(ResponseT, self.request(cast_to, opts, stream=stream, stream_cls=stream_cls))
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\openai\_base_client.py", line 969, in request
    response = self._client.send(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 922, in send
    response.read()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 881, in read
    self._content = b"".join(self.iter_bytes())
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 897, in iter_bytes
    for raw_bytes in self.iter_raw():
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_models.py", line 951, in iter_raw
    for raw_stream_bytes in self.stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_client.py", line 153, in __iter__
    for chunk in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpx\_transports\default.py", line 127, in __iter__
    for part in self._httpcore_stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\connection_pool.py", line 403, in __iter__
    for part in self._stream:
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 334, in __iter__
    for chunk in self._connection._receive_response_body(**kwargs):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 203, in _receive_response_body
    event = self._receive_event(timeout=timeout)
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_sync\http11.py", line 217, in _receive_event
    data = self._network_stream.read(
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\httpcore\_backends\sync.py", line 128, in read
    return self._sock.recv(max_bytes)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1263, in recv
    return self.read(buflen)
  File "C:\Program Files\Python311\Lib\ssl.py", line 1136, in read
    return self._sslobj.read(len)

2025-05-19 18:33:29,007 INFO: [hr_assign_role] Відповідь HR ШІ: {'text': '\\boxed{```json\n{\n  "1715400000": "add_role 658217734814957578 1374046498534330428"\n}\n```}', 'has_json': True, 'json_data': {'1715400000': 'add_role 658217734814957578 1374046498534330428'}}
2025-05-19 18:33:29,011 INFO: [hr_assign_role] Завдання HR ШІ додано до scheduled_commands.
2025-05-19 18:33:29,016 INFO: Websocket closed with 1000, cannot reconnect.
2025-05-19 18:33:29,016 ERROR: Attempting a reconnect in 3.39s
Traceback (most recent call last):
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\client.py", line 617, in connect
    await self.ws.poll_event()
  File "C:\Users\minesko\AppData\Roaming\Python\Python311\site-packages\discord\gateway.py", line 630, in poll_event
    raise ConnectionClosed(
discord.errors.ConnectionClosed: Shard ID None WebSocket closed with 1000
2025-05-19 18:33:29,307 INFO: Роль `1374046498534330428` додано користувачу `658217734814957578`.
2025-05-19 18:33:32,680 INFO: Shard ID None has sent the RESUME payload.
2025-05-19 18:33:32,818 INFO: Shard ID None has successfully RESUMED session d0b75191606861d699ab04d075b6bc47 under trace ["gateway-prd-us-east1-d-l1gp",{"micros":1718,"calls":["id_created",{"micros":0,"calls":[]},"session_lookup_time",{"micros":928,"calls":[]},"session_lookup_finished",{"micros":12,"calls":[]},"discord-sessions-prd-2-3",{"micros":38}]}].
2025-05-19 18:33:43,772 INFO: Received signal to terminate bot and event loop.
2025-05-19 18:33:43,772 INFO: Cleaning up tasks.
2025-05-19 18:33:43,772 INFO: Cleaning up after 3 tasks.
2025-05-19 18:33:43,773 INFO: All tasks finished cancelling.
2025-05-19 18:33:43,773 INFO: Closing the event loop.
